cmd/hub/cli/cli.go

```diff
@@ -38,12 +38,12 @@ var (
 				Key:      "org",
 				DefValue: "",
 			},
-			"key": {
-				Key:      "key",
+			"apiKey": {
```

Comment:
I realized these new hub configuration fields / flags conflict with the bucket `key` configuration flag. They were added in `v2.1.4`... but I'm guessing it hasn't been an issue since the bucket flags are applied after the hub flags (and buckets wouldn't work at all if `key` wasn't reachable from the local config).